sql-cli/sql_cli/__main__.py

```diff
@@ -139,17 +140,20 @@ def run(
     set_airflow_database_conn(airflow_meta_conn)
 
     try:
-        dag_file = generate_dag(
-            directory=project.directory / project.workflows_directory / workflow_name,
-            dags_directory=project.airflow_dags_folder,
-        )
+        if gen_dag:
+            dag_file = generate_dag(
+                directory=project.directory / project.workflows_directory / workflow_name,
+                dags_directory=project.airflow_dags_folder,
+            )
+            dag = get_dag(dag_id=workflow_name, subdir=dag_file.parent.as_posix(), include_examples=False)
+        else:
+            dag = render_dag(directory=project_dir / "workflows" / workflow_name, workflow_name=workflow_name)
```

Comment:
I find the name `render_dag` confusing. What if we called this function `get_virtual_dag`?